purchases/src/main/kotlin/com/revenuecat/purchases/SyncPendingTransactionsHelper.kt

```diff
@@ -31,39 +31,34 @@ internal class SyncPendingTransactionsHelper(
             onSuccess?.invoke(null)
             return
         }
-        if (billing.isConnected()) {
```

Comment:
This is another change I would like to note. This made it so, in the beginning of the SDK initialization, the billing client would likely be disconnected, so it usually failed. The queryPurchases method already has a mechanism to call the callbacks once the billing client is connected, so this seemed unnecesarily. The only thing that slightly concerns me is that this method may take longer now, since it needs to wait for the billing client to connect during SDK initialization. I thought it was ok though. Thoughts?